src/monodroid/jni/debug.cc

```diff
@@ -30,9 +30,9 @@
 #include "java-interop-util.h"
 
 #include "monodroid.h"
-#include "debug.h"
-#include "util.h"
-#include "globals.h"
+#include "debug.hh"
```

Comment:
Why rename some files to be `.hh`, but not others such as `monodroid.h`?  Why rename at all?

Reply:
The `.hh` files contain C++ classes/syntax. This is for the benefit of text editors which use the extension to determine what type of file is being edited (it is also the case with some LSP servers). It follows the convention of  naming `C++` files with the `.cc` or `.cpp` convention. Without renaming the files, some editors/servers would treat the files as `C` code thus greying out the `C++` code (where `#ifdef __cplusplus` is used, for instance) and/or show syntax errors.